### PR TITLE
change ansible environment to 2.3 release

### PR DESCRIPTION
**Pull request by GregMeno**
Signed-off-by: Gregory Meno <gmeno@redhat.com>

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {jewel,kraken,rhcs}-{ansible2.2}-{xenial_cluster,journal_collocation,centos7_cluster,dmcrypt_journal,dmcrypt_journal_collocation,docker_cluster,purge_cluster,purge_dmcrypt,docker_dedicated_journal,docker_dmcrypt_journal_collocation,update_dmcrypt,update_cluster,cluster}
+envlist = {jewel,kraken,rhcs}-{ansible2.3}-{xenial_cluster,journal_collocation,centos7_cluster,dmcrypt_journal,dmcrypt_journal_collocation,docker_cluster,purge_cluster,purge_dmcrypt,docker_dedicated_journal,docker_dmcrypt_journal_collocation,update_dmcrypt,update_cluster,cluster}
```
Comment (andrewschoen) [inline tox.ini]: You'd need to do ``{ansible2.2,ansible2.3}`` so we could still test against both versions.

**Comment by ktdreyer** [comment]
See also https://github.com/ceph/ceph-ansible/pull/1592

**Comment by ktdreyer** [comment]
We need to test both 2.2 and 2.3, until RH can ship 2.3 to customers.

**Comment by guits** [comment]
jenkins test this please

**Pull request merged**